neomodel/relationship_manager.py

```diff
@@ -453,25 +509,44 @@ class ZeroOrMore(RelationshipManager):
     """
     A relationship of zero or more nodes (the default)
     """
-    description = "zero or more relationships"
-
-
-def _relate(cls_name, direction, rel_type, cardinality=None, model=None):
-    if not isinstance(cls_name, (basestring, object)):
-        raise ValueError('Expected class name or class got ' + repr(cls_name))
 
-    if model and not issubclass(model, (StructuredRel,)):
-        raise ValueError('model must be a StructuredRel')
-    return RelationshipDefinition(rel_type, cls_name, direction, cardinality, model)
-
-
-def RelationshipTo(cls_name, rel_type, cardinality=ZeroOrMore, model=None):
-    return _relate(cls_name, OUTGOING, rel_type, cardinality, model)
-
-
-def RelationshipFrom(cls_name, rel_type, cardinality=ZeroOrMore, model=None):
-    return _relate(cls_name, INCOMING, rel_type, cardinality, model)
+    description = "zero or more relationships"
 
 
-def Relationship(cls_name, rel_type, cardinality=ZeroOrMore, model=None):
-    return _relate(cls_name, EITHER, rel_type, cardinality, model)
+class RelationshipTo(RelationshipDefinition):
```

Comment:
This is probably slightly faster, during definition but the two are equivalent. 
Perhaps more importantly: Let's start documenting these classes, it is probably the number one cause of why there are diversions between the docs and the code.